fog/types/src/view.rs

```diff
@@ -539,7 +539,9 @@ impl FogTxOut {
     ///
     /// Returns:
     /// * TxOut,
+    ///
     /// Or
+    ///
     /// * An error if recovery failed
```

Comment:
⛏️ This was poorly formatted previously with an `Or` between 2 bulleted list items
I think this portion of the comment is useless, it doesn't provide any more information than the return type `Result<TxOut, FogTxOutError>`
```suggestion
```
Other options are a normal list
```rust
    /// Returns:
    /// * TxOut
    /// * An error if recovery failed
```
spelling it out
```rust
    /// Returns:
    /// TxOut on success or an error if recovery failed
```

Reply:
Might consider it for the other functions that have
But I'm also fine if these stay as they are, they're no worse then they were previously
```rust
/// * blah
///
/// Or
///
/// * baz
```

Reply:
I didn't really like adding in the extra lines, but it was finding something that the doc formater liked that kept the original intent.